tests/src/test/java/alluxio/job/migrate/MigrateIntegrationTest.java

```diff
@@ -103,7 +105,8 @@ public void migrateDirectory() throws Exception {
    * Creates a file with the given name containing TEST_BYTES.
    */
   private void createFileWithTestBytes(String filename) throws Exception {
-    try (FileOutStream out = mFileSystem.createFile(new AlluxioURI(filename))) {
+    try (FileOutStream out = mFileSystem.createFile(new AlluxioURI(filename),
+        CreateFilePOptions.newBuilder().setWriteType(WritePType.CACHE_THROUGH).build())) {
```

Comment:
The convenient thing about writing the file persisted, is that it will automatically make all the ancestor inodes persisted as well. Using `LocalAlluxioClusterResource.Config` would mean I would have to add that to both test cases?